src/components/GeneralInfo/dataMapper/dataMapper.test.js

```diff
@@ -38,7 +36,51 @@ it('diskMapper', () => {
         mounttype: 'type',
       },
     ])
-  ).toMatchSnapshot();
+  ).toMatchInlineSnapshot(`
```

Comment:
We could instead of using an inline snapshot query the specific cells that should be shown with these arguments passed in. Maybe there is a list somewhere that his can even be based on and made dynamic in order to test any variation?

Reply:
I think these tests are already straightforward and check the return value for each function. I am not sure querying some fields will improve the tests much. Though, I will leave a TODO note to try to implement dynamic tests and cover more cases

Reply:
It's convenient to use inline snapshots in this scenario when you assert data structures. If the implementation changes, the tests will catch the difference and if it's anticipated, it's very quick to update the snapshots.